projects/models.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  pkgs,
+  sources,
+}:
+let
+  yants = import sources.yants { };
+
+  inherit (yants)
+    string
+    list
+    option
+    attrs
+    ;
+in
+rec {
+  project =
+    p: with p; {
+      name = string name;
+      metadata = with metadata; {
+        summary = string summary;
+        fund = string fund;
+        status = string status;
```

Comment:
is this something like packaged/not yet packaged/not planned? Could be an enum then.

Reply:
There's a status field in the NLnet dashboard, but that's not very useful for us here, and really just an accounting thing. It's supposed to say something about handling status by partners. We keep this in our internal system already and that's good enough.